automation/test.sh

```diff
@@ -200,14 +200,12 @@ EOF
 
 if [ "${CLUSTERENV}" == "$ocenv" ]
 then
-    export VALIDATOR_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt-template-validator/releases | \
+    # Deploy ssp-operator
+    export SSP_VERSION=$(curl -s https://api.github.com/repos/kubevirt/ssp-operator/releases | \
             jq '.[] | select(.prerelease==false) | .tag_name' | sort -V | tail -n1 | tr -d '"')
-
-    git clone -b ${VALIDATOR_VERSION} --depth 1 https://github.com/kubevirt/kubevirt-template-validator kubevirt-template-validator
-    VALIDATOR_DIR="kubevirt-template-validator/cluster/ocp4"
-    sed -i 's/RELEASE_TAG/'$VALIDATOR_VERSION'/' ${VALIDATOR_DIR}/service.yaml
-    oc apply -n kubevirt -f ${VALIDATOR_DIR}
-    oc wait --for=condition=Available --timeout=${timeout}s deployment/virt-template-validator -n $namespace
+    git clone -b ${SSP_VERSION} --depth 1 https://github.com/kubevirt/ssp-operator ssp-operator
+    make -C ssp-operator deploy
```

Comment:
Can you make sure this will not enable deployment of `common-templates` in SSP? IIRC in its default configuration it will enable the deployment which could clash with the common-templates deployed by this test.

Reply:
AFAIR templates from this repo are deployed in kubevirt namespace, so there should be no conflict in templates. 
I would not clone the ssp repo, but deploy it directly from manifests, because this clone will anyway deploy only released version

Reply:
@ksimon1: done